bpf/lbmap/lbmap.go

```diff
@@ -36,13 +36,11 @@ import (
 	"github.com/cilium/cilium/common/addressing"
 	"github.com/cilium/cilium/common/bpf"
 	"github.com/cilium/cilium/common/types"
-	"github.com/op/go-logging"
 
 	"github.com/codegangsta/cli"
 )
 
 var (
-	log  = logging.MustGetLogger("cilium-net")
```

Comment:
I thought this would affect the API, for example that other function would call `lbmap` package and the errors wouldn't be logged.